ci-operator/config/ibmz-eco/isv-operator/ibmz-eco-isv-operator-master.yaml

```diff
@@ -0,0 +1,38 @@
+binary_build_commands: make install
+build_root:
```

Comment:
This also should not exist since you are not building any source or output images from a repo